src/views/Create.vue

```diff
@@ -106,22 +110,28 @@
 <script>
 import { mapActions } from 'vuex';
 import namespaces from '@/namespaces.json';
+import draggable from 'vuedraggable';
 
 export default {
+  components: {
+    draggable
+  },
   data() {
     return {
       key: this.$route.params.key,
       loading: false,
+      choices: [],
```

Comment:
Why not using `form.choice`?

Reply:
1st issue was because I needed to have an unique `:key` in the for loop so  the transition would perform on the correct element , hence i had to change `form.choices` from text to object array with key attribute.

Next was when i tried mapping `form.choices` from object to a number array to prepare for submission, `isValid()` would always trigger and throw an error. So in the end i decided to use a different array to avoid the  `isValid()` error.